.github/workflows/tests.yaml

```diff
@@ -0,0 +1,64 @@
+
+name: Tests
+on:
+  pull_request:
+  push:
+    paths:
+      - "action.yaml"
+      - ".github/test_files/**"
+    branches:
+      - "main"
+    tags: ["v*.*.*"]
+
+jobs:
+  k8s-manifest-tests:
+    runs-on: ubuntu-latest
+    name: Test k8s manifest scanning
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Scan k8s manifest
+        uses: ./
+        with:
+          service_account_credentials: ${{ secrets.MONDOO_SECRET }}
```

Comment:
Instead of MONDOO_SECRET, should we follow our own docs and use 'MONDOO_CLIENT_ACCOUNT'?

Reply:
Renamed the secret